### ISSUE TITLE
Add option to use 'select' (drop down) as a field type

### ISSUE THREAD

**Issue by mjamesderocher** [dennispassway/sanity-plugin-media-library#17]
I think the new ability to add custom fields is great, but it would really help me if we could also use a select element as a field. I want to manage image rights types (copyright, public domain, Creative Commons, etc), and it would be great if I could select those from a dropdown instead of having to remember the right value to type in each time.

**Comment by Aratramba** [comment]
sounds like a sensible option to have. We'll look into it.

**Comment by Aratramba** [comment]
this was released in 1.0.6 and can be [configured](https://github.com/dennispassway/sanity-plugin-media-library#configuration--theming) like this:

```
{
      "label": "Copyright",
      "name": "copyright",
      "placeholder": "pick one…",
      "type": "select",
      "options": [
        { "title": "Copyright", "value": "copyright" },
        { "title": "Public Domain", "value": "public-domain" },
        { "title": "Creative Commons", "value": "creative-commons" }
      ]
},
```

<img src="https://user-images.githubusercontent.com/580312/106911854-e5ec5b80-6702-11eb-91c4-bde8a618bafc.png" width="394" />
